src/oidc/validate.ts

```diff
@@ -16,13 +16,7 @@ limitations under the License.
 
 import { IClientWellKnown, IDelegatedAuthConfig, M_AUTHENTICATION } from "../client";
 import { logger } from "../logger";
-
-export enum OidcDiscoveryError {
```

Comment:
This looks like a breaking change, can we have a re-export to maintain backwards compatiblity?